origin-graphql/package.json

```diff
@@ -21,22 +21,22 @@
     "prettier:check": "prettier -c *.js \"src/**/*.js\" \"test/**/*.js\""
   },
   "dependencies": {
-    "@babel/core": "^7.2.2",
+    "@babel/core": "^7.3.4",
     "@babel/register": "^7.0.0",
-    "apollo-link-context": "^1.0.14",
-    "apollo-link-http": "^1.5.10",
-    "dayjs": "^1.8.5",
+    "apollo-link-context": "^1.0.15",
+    "apollo-link-http": "^1.5.12",
+    "dayjs": "^1.8.10",
     "json-stable-stringify": "^1.0.1",
     "json-stable-stringify-without-jsonify": "^1.0.1",
```

Comment:
Can we remove this? I added the plain `json-stable-stringify` in a recent PR because it was missing/trying to import it.

Reply:
Sure, removed it